docs/content/preview/reference/drivers/go/yb-pgx-reference.md

```diff
@@ -0,0 +1,308 @@
+      PGX Driver
+    </a>
+  </li>
+
+  <li >
+    <a href="/preview/reference/drivers/go/pq-reference/" class="nav-link">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      PQ Driver
+    </a>
+  </li>
+
+</ul>
+
+[YugabyteDB PGX driver](https://github.com/yugabyte/pgjdbc) is a Go driver for [YSQL](/preview/api/ysql/) based on [PGX driver](https://github.com/jackc/pgx/).
+
+## Quick start
+
+Learn how to establish a connection to YugabyteDB database and begin CRUD operations using the steps from [Build a Go application](../../../../quick-start/build-apps/go/ysql-yb-pgx/).
+
+This page provides details for getting started with [YugabyteDB PGX driver](https://github.com/yugabyte/pgx) for connecting to YugabyteDB YSQL API.
+
+## Import the driver package
+
+You can import the YugabyteDB PGX driver package by adding the following import statement in your Go code.
+
+```go
+import (
+  "github.com/yugabyte/pgx/v4"
+)
+```
+
+Optionally, you can choose to import the pgxpool package instead. Refer to [Using pgxpool API](#using-pgxpool-api) to learn more.
+
+## Fundamentals
+
+Learn how to perform common tasks required for Go application development using the YugabyteDB PGX driver.
+
+### Connect to YugabyteDB database
+
+After setting up the driver, implement the Go client application that uses the driver to connect to your YugabyteDB cluster and run a query on the sample data.
+
+The YugabyteDB PGX driver allows Go programmers to connect to YugabyteDB to execute DMLs and DDLs using the PGX APIs. It also supports the standard `database/sql` package.
+
+Use the `pgx.Connect()` method or `pgxpool.Connect()` method to create a connection object for the YugabyteDB database. This can be used to perform DDLs and DMLs against the database.
+
+The driver has the following features:
+
+- It is **cluster-aware**, which eliminates the need for an external load balancer.
+
+Connections are distributed across all the [YB-Tservers](/preview/architecture/concepts/yb-tserver/) in the cluster, irrespective of their placements.
+
+The following table describes the connection parameters required to connect to the YugabyteDB database with **Uniform load balancing**.
+
+| Parameters | Description | Default |
+| :---------- | :---------- | :------ |
+| host  | hostname of the YugabyteDB instance | localhost
+| port |  Listen port for YSQL | 5433
+| database | Database name | yugabyte
+| user | User connecting to the database | yugabyte
+| password | Password for the user | yugabyte
+| load_balance | enables uniform load balancing | true
+
+To enable the cluster-aware connection load balancing, provide the parameter `load_balance=true` in the connection string as follows:
+
+```go
+"postgres://username:password@localhost:5433/database_name?load_balance=true"
+```
+
+- It is **topology-aware**, which is essential for geographically-distributed applications.
+
+The following is a code snippet for connecting to YugabyteDB using the connection parameters.
```

Comment:
Lines from 98 till 105 should be moved somewhere else, may be after line# 94 and before `It is **topology-aware** ...`. These are not related to topology-aware load balancing.